tests/test_rpi_models.py

```diff
@@ -25,7 +25,7 @@ def test_model_A(self):
         self.assertIn(getrpi('0008'), modela)
         self.assertIn(getrpi('0009'), modela)
 
-    def test_model_A_plus(self):
+    def test_module_a_plus(self):
```

Comment:
Did you mean to do this? If so, sureley `test_model_B` and `test_model_B_plus` should be changed too.